### ISSUE TITLE
Categories of mobile roadmap

### ISSUE THREAD

**Issue by dontcallmedom** [w3c/web-roadmaps#43]
Another random idea of how to structure the mobile roadmap - reuse the categories in `https://html5test.com/` ("Semantics", "Device Access", …) or find out where they are coming from in the first place :)

**Comment by xfq** [comment]
Yet another random source: https://platform.html5.org/

**Comment by xfq** [comment]
@tidoust Can we close this?

**Comment by tidoust** [comment]
Yes.

For archival, we reviewed different platforms and categorizations schemes. There is some overlap in all of them (e.g. storage), and differences that seem somewhat arbitrary. For instance, the [Web Platform](https://platform.html5.org/) has categories such as "Other core-platform bonus features" and "Various others" which do not mean much. Similarly, [HTML5test](https://html5test.com/) has "Security" under "performance & integration", which seems strange.

In the end, we decided to stick to what we had already. It may not be perfect, but:

- categories seem meaningful enough
- it's a one-level categorization scheme, which is helpful to keep the structure simple
- we now have icons for each category

Two minor changes we made:

1. the "Media and RTC technologies" category is now simply "Media"
2. the "Network" category is now "Network and Communications", and features WebRTC.
